[TASK] Add separate workflow for composer scripts (#258)
TYPO3-Documentation/tea: README.md

## Running tests locally via DDEV

### Unit tests

To run unit tests, type:

```bash
composer ci:tests:unit
```

## Running lints in CI
### GitHub

> For GitHub, we prepared two ways of running lints. First one relies on executing

Reviewer: ```suggestion
For GitHub, we prepared two ways of running lints:
```

> composer scripts, and second is using exported, prepared GitHub Actions in your

Reviewer: ```suggestion
1. relies on executing composer scripts
2. second is using exported, prepared GitHub Actions in your workflow.
```

> workflow.

Reviewer: ```suggestion
```

#### Composer scripts

> You can run your lints using composer scripts. Example workflow is defined in

Reviewer: ```suggestion
You can run your lints using composer scripts. An example workflow is defined in
```

`ci-composer-scripts.yml`.
#### Ready to use GitHub Actions
You can use prepared GitHub Actions. All of ready to use GitHub Actions are in
[TYPO3 Continuous Integration organisation](https://github.com/TYPO3-Continuous-Integration).

> Example workflow is defined in `ci.yml`.

Reviewer: ```suggestion
An example workflow is defined in `ci.yml`
```

### GitLab

> For GitLab, please use pipeline that is defined in `.gitlab-ci.yml`.

Reviewer: ```suggestion
For GitLab, please use the pipeline that is defined in `.gitlab-ci.yml`.
```
